javascript/chronometer.js

```diff
@@ -1,26 +1,68 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
+    this.msIntervalId = 0;
+    this.msTime = 0;
   }
+
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (callback) { callback(); }
+    }, 1000);
   }
+
+  msInterval(callback) {
+    this.msIntervalId = setInterval(() => {
+      if (this.msTime === 99) { this.msTime = 0; }
+      else { this.msTime++; }
+
+      if (callback) { callback(); }
+    }, 10);
+  }
+
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60);
   }
+
   getSeconds() {
-    // ... your code goes here
+    return this.currentTime - (this.getMinutes() * 60);
```

Comment:
smart approach. Another way to go around this with the % operator for the reminder. 
`this.currentTime % 60`